apps/base-docs/tutorials/docs/0_gassless-transactions-with-paymaster.md

```diff
@@ -0,0 +1,616 @@
+
+## Objectives
+
+- **Understand Security Policies:** Learn how to configure security measures to ensure safe and reliable transactions.
+- **Implement Budget Practices**: Discover effective ways to manage and allocate resources for sponsored transactions.
+- **Sponsor User Transactions:** Gain the ability to subsidize transaction fees for users, enhancing the user experience by making transactions free for them.
+- **Enable Scheduled Sponsorships:** Explore methods to set up and manage sponsored transactions on various schedules, including weekly, monthly, and daily cadences.
+
+## Prerequisites
+
+This tutorial assumes you have a Coinbase Cloud Developer Platform account. If not, sign up [here](https://portal.cdp.coinbase.com/).
+
+- **Coinbase CDP account:** This is your access point to the Coinbase Cloud Developer Platform, where you can manage projects and utilize tools like the Paymaster.
+- **Familiarity with Smart Accounts and ERC 4337:** Understanding Smart Accounts and the ERC 4337 standard is crucial as they are the backbone of executing advanced transaction patterns and account abstractions on the Ethereum network.
+- **Rust:** Rust is a systems programming language focused on safety and performance. You will need basic knowledge of Rust for some coding aspects of this tutorial.
+- **Foundry:** Foundry is a development environment, testing framework, and smart contract toolkit for Ethereum. It's essential for deploying and testing smart contracts.
+
+### Setup a Coinbase Paymaster & Bundler
+
+In this section, you will configure a Paymaster to sponsor payments on behalf of a specific smart contract for a specified amount. First, navigate to the Coinbase Developer Platform, create or select your project, and click on the Paymaster tool from the left navigation. Then, go to the Configuration tab and save the RPC URL to your clipboard, which will be needed for later steps in your index.js file.
+
+**Navigate to the** **[Coinbase Developer Platform](https://portal.cdp.coinbase.com/)**:
+
+Create or select your project of choice from the upper left corner of your screen.
+
+![cdp-home.png](../../assets/images/gasless-transaction-on-base/cdp-select-project.png)
+
+**Click on the `Paymaster` tool on the left navigation:** [Paymaster Tool](https://portal.cdp.coinbase.com/products/bundler-and-paymaster)
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-paymaster.png)
+
+**Click on `Configuration` at the top of the screen**
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-config.png)
+
+**Save the RPC URL to your paymaster to your clipboard:** You will need it in your `index.js` file in a later step.
```

Comment:
Nit, but I'd call this the `Paymaster & Bundler endpoint` to match the text on the site.

Reply:
all good